Packages/doc/labnotebook-docs.rst

```diff
@@ -99,10 +99,12 @@ Caveats
 * Some entries are, for historical reasons, present in both the first layer and
   the headstage independent layer (nineth layer). New code should query the data
   from the headstage independent layer only.
-* In case unassociated DA/AD channels were used during acquisition there are
-  additional entries present which are formatted like `$entry UNASSOC_$channelNumber`
-  and only have entries in the nineth layer (as it is by definition headstage
-  independent data).
+* In case unassociated DA/AD channels were used (these are channels which are
+  not connected to a headstage) during acquisition there are additional entries
+  present which are formatted like `$entry UNASSOC_$channelNumber` and only
+  have entries in the nineth layer (as it is by definition headstage
```

Comment:
@t-b 
typo: ninth

I'm confused because the unassociated channel number in the experiment we looked at was a single digit. Would you help me out and show what unassocated DA0 and AD0 look like?

Reply:
We can discuss that today.